skorch/tests/test_net.py

```diff
@@ -2138,14 +2138,22 @@ def test_batch_count(self, net_cls, module_cls, data, batch_size):
         assert net.history[:, "train_batch_count"] == [train_batch_count]
         assert net.history[:, "valid_batch_count"] == [valid_batch_count]
 
+    @flaky(max_runs=3)
```

Comment:
Does increasing `max_epochs` make this less flaky?

Reply:
The test is not actually flaky. I just added it because I thought it to be cautious for any test where we expect the model to reach a certain score.

My general impression is that tests have become less flaky, perhaps because of the `seeds_fixed` auto-fixture. Do you believe we should remove the flaky decorators?

Reply:
It would be nice to have a `seeds_fixed`. Although this is not a blocker for this PR.

Reply:
We have it already here:

https://github.com/skorch-dev/skorch/blob/9fafede929891e1c5c98ba4e7764035b75a25d26/skorch/tests/conftest.py#L23-L27

Reply:
I was just about to implement this and saw it. You added it not too long ago. :) This should mean our tests are deterministic. (hopefully)